### PR TITLE
what I think needs to be changed to go public.

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

**Comment by ldecicco-USGS** [comment]
@lindsaycarr here's what I think we need to do for each of the pages.

File: content/intro-curriculum/A_Introduction.Rmd [hunk 1]
```diff
@@ -3,14 +3,15 @@ title: "A. Introduction to R"
...
+image: "img/main/intro-icons-300px/r-logo.png"
 output: USGSmarkdowntemplates::hugo
 ---
 
 ```{r setup, include=FALSE, warning=FALSE, message=FALSE}
 library(knitr)
 
 knit_hooks$set(plot=function(x, options) {
-  sprintf("<img src='../%s%s-%d.%s'/ alt='/%s'/>", 
+  sprintf("<img src='../%s%s-%d.%s'/ title='/%s'/>",
```
Comment (lindsaycarr) [inline content/intro-curriculum/A_Introduction.Rmd]: @ldecicco-USGS this did not add the title as hover text. Is that what we expected?

Comment (ldecicco-USGS) [inline content/intro-curriculum/A_Introduction.Rmd]: are there any figures generated by R on this page?

File: content/intro-curriculum/H_Visualize.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ qtemp_gg <- ggplot(data=intro_df, aes(x=Flow_Inst, y=Wtemp_Inst))
 qtemp_gg
 ```
 
-<img src='../static/ggplot2/ggplot_examp-1.png'/ alt='/Blank ggplot2 plot'/>
+<img src='../static/ggplot2/ggplot_examp-1.png'/ title='/Blank ggplot2 plot'/>
```
Comment (ldecicco-USGS) [inline content/intro-curriculum/H_Visualize.md]: So here's an example where there are R generated figures, and all I really did was update that header chunk.

File: content/intro-curriculum/A_Introduction.Rmd (hunk 1 above)
Comment (ldecicco-USGS) [inline content/intro-curriculum/A_Introduction.Rmd]: To add titles to regular links:

```
This is [an example](http://example.com/ "Title")
```

Comment (lindsaycarr) [inline content/intro-curriculum/A_Introduction.Rmd]: yes, there are pictures

Comment (lindsaycarr) [inline content/intro-curriculum/A_Introduction.Rmd]: oh, but not figures. Just static images. I see what I have to do now

**Pull request closed without merging**